lib/app/backend/CliProxy.js

```diff
@@ -120,7 +146,7 @@ class CliProxy {
       }
 
       const headers = {
-        'sg-application-id': `shop_${shopNumber}`,
+        'sg-application-id': `admin_${shopNumber}`,
```

Comment:
This is a mistake. The shopNumber variable contains already the whole application it.
You should rename the variable to `applicationId` and pass it without any manipulation to the header.

Reply:
Its also wrong in line 97

Reply:
Changed